lib/Server.js

```diff
@@ -389,6 +389,23 @@ function Server(compiler, options) {
 			const pems = selfsigned.generate(attrs, {
 				algorithm: "sha256",
 				days: 30,
+				extensions: [{
+					name: "subjectAltName",
+					altNames: [
+						{
+							type: 2, // DNS
+							value: "localhost"
+						},
+						{
+							type: 6, // URI
+							value: "localhost"
+						},
+						{
+							type: 7, // IP
+							ip: "127.0.0.1"
+						}
+					]
```

Comment:
While we're making changes to this, consider adding IPv6 loopback IPs

`::1`  Most browsers are ok with this being a DNS entry type, but some versions of chrome want it as an IP
`fe80::1`